storage/storage-service/src/storage_service_test.rs

```diff
@@ -57,19 +48,15 @@ proptest! {
     #[test]
     fn test_storage_service_basic(blocks in arb_blocks_to_commit().no_shrink()) {
         let(_tmp_dir, _server_handler, read_client, write_client) =
-            start_test_storage_with_read_write_client(/* need_to_use_genesis = */ true);
+            start_test_storage_with_read_write_client();
 
         let mut version = 0;
         let mut all_accounts = BTreeMap::new();
 
-        // Add the genesis account to the list of all accounts; we will validate
-        // it when testing the backup API.
-        all_accounts.insert(GENESIS_INFO.3.hash(), GENESIS_INFO.4.clone());
```

Comment:
GENESIS_INFO.3 and GENESIS_INFO.4 can also be removed.  Maybe we can get rid of GENESIS_INFO static entirely?